### ISSUE TITLE
Add0n.com's use of Cloudflare is very unfriendly

### ISSUE THREAD

**Issue by huyz** [rNeomy/reader-view#142]
You're probably not aware, but all the links on https://chrome.google.com/webstore/detail/reader-view/ecabifbgmdmgdllomnfinbmaellmclnh?hl=en point to add0n.com, which is a huge problem for legitimate users.
This site's Cloudflare access policies are extremely aggressive and block out tons of legitimate home users who have the misfortune of sitting behind a CGNAT.

This is what I get:
![screenshot 2022-08-07T050332Z](https://user-images.githubusercontent.com/128394/183276091-21076ec4-78fc-4a03-8bfe-83c43b4065ca.png)

This is almost unique to add0n.com, as I can't remember the last time I've been blocked like this.

This is of course very bad for users of the Chrome extension who cannot visit the web page or even find the github link.

At the very least, there should be a direct github.com link at the bottom of the description at https://chrome.google.com/webstore/detail/reader-view/ecabifbgmdmgdllomnfinbmaellmclnh?hl=en

**Comment by huyz** [comment]
I get a new tab with this "Access denied" every time the extension self-updates (e.g., https://add0n.com/chrome-reader-view.html?version=0.6.3&p=0.5.4&type=update)

Please, do NOT use add0n.com anymore

**Comment by rNeomy** [comment]
What is your IP address? This is Cloudflare that is blocking your IP address, not the FAQs page.

**Comment by rNeomy** [comment]
@huyz, the links on Chrome WebStore now point to the GitHub page.

**Comment by huyz** [comment]
Yeah I realize that it's Cloudflare blocking, but isn't there a way to customize the level of sensitivity? It's just strange that I am blocked from add0n.com but no other Cloudflare website.  Add0n.com doesn't sound like a very trafficked website, so I'm not sure why Cloudflare would decide to protect it from DDoS so aggressively.

My IP address: https://privnote.com/LTUCkzN5#LGwTJycMk

**Comment by rNeomy** [comment]
Thanks! 

I'll try to follow up to see if we can fix the issue. Meanwhile, use the options page to disable the FAQs page opening on updates.

**Comment by huyz** [comment]
@rNeomy Hi again, I just signed up for a fixed IP address; i.e. an IP that isn't shared with a group of residential customers. Yet I'm still blocked.  Maybe my entire country or region is blocked by your site?

The Cloudflare error page says "Provide the site owner this information".

So here it is if you want to check:
https://privnote.com/aaRgvLiz#DXFwyUamP
